spec/toggle-quotes-spec.js

```diff
@@ -73,6 +73,17 @@ describe('ToggleQuotes', () => {
       })
     })
 
+    describe('when using a scope-specific config override', () => {
+      it('prefers the scope-specific setting', () => {
+        atom.config.set('toggle-quotes.quoteCharacters', '\'"~', { scopeSelector: '.source.js' })
+        editor.setCursorBufferPosition([0, 16])
+        toggleQuotes(editor)
+        expect(editor.lineTextForBufferRow(0)).toBe('console.log(~Hello World~)')
```

Comment:
I'd like an extra assertion to make sure that this is _not_ the case when the file's scope is not `source.js`.

Reply:
Ah, good point.